### PR TITLE
feature/viewing-my-tasks-mvp

### PR DESCRIPTION
**Pull request by chopkinsmade**
## Description of change

This PR merges the feature branch for the my tasks tab on the dashboard into main. The tickets here have already been merged with 3 separate PRs. (https://github.com/uktrade/data-hub-frontend/pull/6257, https://github.com/uktrade/data-hub-frontend/pull/6288 and https://github.com/uktrade/data-hub-frontend/pull/6311)

A Tasks tab will now show on the homepage **provided** a user has one or more Tasks. The Investment projects summary has moved to the Investment projects tab.

## Test instructions
When no Tasks are created the Tasks tab should not be visible on the homepage.

When Tasks have been created they should be visible in the Tasks tab on the homepage.

Investment projects summary should now be visible on the Investment projects tab.

### Before
image

#### Investment projects tab
image

#### Tasks tab
image

_Add a screenshot_

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [x] Has the branch been rebased to main?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: src/client/components/Dashboard/my-tasks/MyTasksTable.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      <Table.Cell setWidth="15%">{formatMediumDate(task.due_date)}</Table.Cell>
+      <Table.Cell setWidth="40%">
+        <Link
+          href={urls.tasks.details(task.id)}
+          data-test={`${task.id}-task-link`}
+        >
+          {task.title}
+        </Link>
+      </Table.Cell>
+      <Table.Cell setWidth="20%">{task.investment_project.name}</Table.Cell>
```
Comment (CaitBarnard) [inline src/client/components/Dashboard/my-tasks/MyTasksTable.jsx:39]: We need this to be `task.investment_project?.name` as it errors when running the local API which has generic tasks not attached to an investment project

File: src/client/components/DashboardTabs/index.jsx [hunk 2]
```diff
@@ -37,6 +39,13 @@ const DashboardTabs = ({
       keepQueryParams={false}
       onTabChange={onTabChange}
       tabs={{
+        ...(hasTasks && {
+          [urls.dashboard.myTasks()]: {
+            label: 'My tasks',
+            content: <MyTasks />,
+            content: <MyTasks adviser={adviser} />,
```
Comment (DeanElliott96) [inline src/client/components/DashboardTabs/index.jsx:46]: There is `content` twice here, is that correct?

**Review by DeanElliott96 — APPROVED** [review]
Looks good, maybe an accidental duplicate `content`?

File: src/client/components/DashboardTabs/index.jsx (hunk 2 above)
Comment (CaitBarnard) [reply src/client/components/DashboardTabs/index.jsx]: Nice spot, thanks!

**Review by samuele-mattiuzzo — APPROVED** [review]

**Pull request merged**